Add a few missing locale strings on Avalonia

gdkchan opened the pull request:
Adds two groups of missing strings on the english locale file:
- Graphics scaling filter options (Bilinear, Nearest, FSR).
- Multiplayer network modes (Disabled, LdnMitm).

For the network modes, they were sourced directly from the enum before. To allow them to be translated, I changed the combo box to have its own items that can be translated. I also changed "LdnMitm" to "LDN MitM". Might be worth testing that it is still working properly.

TSRBerry commented:
> I also changed "LdnMitm" to "LDN MitM".

If we are changing this again shouldn't we just use the name of the sysmodule (ldn_mitm)?

gdkchan commented:
> > I also changed "LdnMitm" to "LDN MitM".
> 
> If we are changing this again shouldn't we just use the name of the sysmodule (ldn_mitm)?

Perhaps, it doesn't matter too much for me. I assumed that making it as close to readable text as possible would be desirable, but I can see why one would want to match the sysmodule name exactly (make more obvious that it is compatible with it). If we do that I guess we ultimately wouldn't want the name to be translated.

TSRBerry commented on src/Ryujinx/UI/ViewModels/SettingsViewModel.cs:
```diff
@@ -237,11 +237,6 @@ public AvaloniaList<string> NetworkInterfaceList
             get => new(_networkInterfaces.Keys);
         }
 
-        public AvaloniaList<string> MultiplayerModes
-        {
-            get => new(Enum.GetNames<MultiplayerMode>());
-        }
-
```
Instead of removing the binding, couldn't we do something like this?
```cs
public AvaloniaList<string> MultiplayerModes
{
    get
    {
        AvaloniaList<string> modes = [];

        foreach (var multiplayerMode in Enum.GetNames<MultiplayerMode>())
        {
            if (Enum.TryParse($"MultiplayerMode{multiplayerMode}", out LocaleKeys localeKey))
            {
                modes.Add(LocaleManager.Instance[localeKey]);
                continue;
            }

            // Fallback
            modes.Add(multiplayerMode);
        }

        return modes;
    }
}
```

Probably needs to be changed, so it is only evaluated once, but that could be even better than manually adding entries to the combo list, right?

gdkchan replied:
That could be done, although we don't do that for other enums right now, so it would be a bit inconsistent.

TSRBerry replied:
Oh we don't? Well in that case, this works fine as well.

gdkchan replied:
Yea, the Graphics scaling filter and CPU memory manger mode uses the same approach that I used here. If we changed this one, we should probably change all of them.

TSRBerry commented:
> Perhaps, it doesn't matter too much for me. I assumed that making it as close to readable text as possible would be desirable, but I can see why one would want to match the sysmodule name exactly (make more obvious that it is compatible with it). If we do that I guess we ultimately wouldn't want the name to be translated.

Hmmm, should we poll people about this?

TSRBerry reviewed (approved):
Lgtm! We could still change the "ldn_mitm" name with translations later.

gdkchan commented:
> > I also changed "LdnMitm" to "LDN MitM".
> 
> If we are changing this again shouldn't we just use the name of the sysmodule (ldn_mitm)?

Changed to `ldn_mitm`.
